src/src/AdminBundle/Entity/Registration.php

```diff
@@ -46,7 +46,7 @@ class Registration extends BaseEntity
     /**
      * @var string
      *
-     * @ORM\Column(name="confirmation_token", type="string", length=50)
+     * @ORM\Column(name="confirmation_token", type="string", length=50, unique=true)
```

Comment:
zrovna tu by to problem nemal byt, ale vsade to riesime cez `@UniqueEntity` vid napr riadok 14. v tomto subore. prosim zmenme to tak kvoli konzistencii.